worldedit-core/src/main/java/com/fastasyncworldedit/core/history/changeset/FaweStreamChangeSet.java

```diff
@@ -227,6 +241,10 @@ public void writeHeader(OutputStream os, int x, int y, int z) throws IOException
     public void readHeader(InputStream is) throws IOException {
         // skip mode
         int mode = is.read();
+        int version = is.read();
+        if (version != FaweStreamChangeSet.version) {
+            throw new UnsupportedOperationException(String.format("Version %s history not supported!", version));
```

Comment:
Maybe drop some instructions how to mitigated that?

Reply:
Since we're breaking history compatibility anyway, I figured I'd just add it in now in case it's useful in the future. There'll only be the one version to start with lol